snapcraft/ovn.env

```diff
@@ -7,8 +7,10 @@ if [ -r "$runtime_env" ]; then
 fi
 
 export OVN_PKI_DIR="${SNAP_COMMON}/data/pki"
-export CA_CERT="${OVN_PKI_DIR}/cacert.pem"
-
 export OVN_RUNDIR="${SNAP_COMMON}/run/ovn/"
 export OVN_NB_DB="${OVN_NB_CONNECT}"
 export OVN_SB_DB="${OVN_SB_CONNECT}"
+
+export OVS_RUNDIR="${SNAP_COMMON}/run/switch/"
+
+export CA_CERT="${OVN_PKI_DIR}/cacert.pem"
```

Comment:
1. The declaration of this variable was grouped together with other PKI related items, why did you move it away from them?
2. The change is unrelated to the other changes you make, if you feel strongly about keeping this change please put it into a separate commit as part of this PR with a commit message explaining why you think this is necessary.